### PR TITLE
uprn is now m2m with title

### PR DESCRIPTION
**Pull request by davidread**
* [x] update model
* [x] update read serializer (Andrea?)
* [x] update importer

### CODE REVIEW

File: landavailability/tests/lr/test_models.py [hunk 1]
```diff
@@ -81,8 +81,9 @@ def test_uprn_model_creation(self):
         self.assertEqual(Polygon.objects.count(), 1)
 
         uprn = Uprn()
-        uprn.title = title
         uprn.uprn = "0031535421432"
+        uprn.save()  # we need an id before we can add a M2M relationship
+        uprn.titles.add(title)
```
Comment (andreagrandi) [inline landavailability/tests/lr/test_models.py:86]: This definitely tests if we can associate a Title to a Uprn. We should also include a test to verify the opposite thing: if we can add a Uprn to a Title.

**Review by andreagrandi — CHANGES_REQUESTED** [review]

File: landavailability/tests/lr/test_models.py (hunk 1 above)
Comment (davidread) [reply landavailability/tests/lr/test_models.py:86]: I've added that in now

**Comment by davidread** [comment]
I've updated the importer now. Did you want to do the serializer, or shall I?

**Comment by andreagrandi** [comment]
@davidread yes I can do the serializer. Can you please fix the failing tests first so I can continue from there? Cheers

**Comment by davidread** [comment]
Now I've remembered to add the migration file to the repo, only one test should fail: tests/lr/test_views.py::TestUprnLookupView::test_lr_view_lookup_uprn which is due to the serializer

**Comment by andreagrandi** [comment]
@davidread it should be all fixed now. Coveralls is not very happy at the moment, but we can improve it later :)

**Comment by davidread** [comment]
@andreagrandi I'm good with your bits. Do you want to review the rest and merge if happy?

**Pull request merged**